### PR TITLE
Support for standalone RevoScale package scenario - Windows

### PR DESCRIPTION
**Pull request by arunguru-msft**
Adding standalone revoscale packages description

### CODE REVIEW

**Comment by jborsecnik** [comment]
@WilliamDAssafMSFT, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md [hunk 1]
```diff
@@ -76,8 +76,9 @@ If you encounter any installation errors during setup, check the summary log in
 
 Beginning with [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)], runtimes for R, Python, and Java are no longer shipped or installed with SQL Server setup. Instead, use the following sections to install your custom runtimes and packages. 
 
-### Install R
+### Setup R support
 
+#### Install R runtime
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md:81]:
```suggestion
#### Install R runtime

```

File: docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md [hunk 2]
```diff
@@ -97,7 +98,7 @@ Beginning with [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)], runtimes f
 
     install.packages("https://aka.ms/sqlml/r4.2/windows/RevoScaleR_10.0.1.zip", repos=NULL)
     ```
-
+#### Configure R runtime with SQL Server
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md:101]:
```suggestion

#### Configure R runtime with SQL Server

```

File: docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md [hunk 3]
```diff
@@ -130,8 +131,8 @@ Beginning with [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)], runtimes f
     GO
     ```
 
-### Install Python
-
+### Setup Python support
+#### Install Python runtime
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md:135]:
```suggestion
### Setup Python support

#### Install Python runtime

```

File: docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md [hunk 4]
```diff
@@ -147,7 +148,7 @@ Beginning with [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)], runtimes f
     cd "C:\Program Files\Python310\"
     python -m pip install https://aka.ms/sqlml/python3.10/windows/revoscalepy-10.0.1-py3-none-any.whl
     ```
-
+#### Configure Python runtime with SQL Server
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md:151]:
```suggestion

#### Configure Python runtime with SQL Server

```

File: docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md [hunk 5]
```diff
@@ -238,6 +239,9 @@ Packages that you want to use from SQL Server must be installed in the default l
 
 To install and manage additional packages, you can set up user groups to share packages on a per-database level. Or you can configure database roles to enable users to install their own packages. For more information, see [Install Python packages](../package-management/install-additional-python-packages-on-sql-server.md) and [Install new R packages](../package-management/install-additional-r-packages-on-sql-server.md).
 
+### Standalone RevoScale packages for Python and R runtime
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install-sql-2022.md:242]:
```suggestion
### Standalone RevoScale packages for Python and R runtime

```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]
made minor edits.

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**